floodviz/templates/index.html

```diff
@@ -18,26 +16,27 @@
 		<div id="hydrograph">
 			<svg></svg>
 		</div>
+		<div id="peakflow_bar"></div>
 		<div id="map"></div>
```

Comment:
I think we are going to want the map next to the hydrograph ultimately. Please put the peakflow-bar below the map